### ISSUE TITLE
Add a way to add planters with remote control discord bot

### ISSUE THREAD

**Issue by MilkiBSS** [NatroTeam/NatroMacro#766]
### Have you searched the existing suggestions?

Yes

### What is your suggestion?

That it would be nice to have a way to be able to add planters with the discord remote control bot

### How will this improve Natro Macro?

By giving users a way to do this

### What is your Discord username?

Milkyreborn1470

**Comment by Noobyguy775** [comment]
Already possible. `?planter add`

**Comment by MilkiBSS** [comment]
Doesnt that just add time to an existing planter

**Comment by Noobyguy775** [comment]
oh sorry, you're right, there is not a `?planter` command to add a planter

**Comment by sparkles3421** [comment]
?set [planter name]Check 1

Eg:
?set CandyPlanterCheck 1

I think this will only work with planter+(auto) enabled.

**Comment by MilkiBSS** [comment]
Yeah thats useful but i was more requesting an feature to add an planter from scratch to the timers revision tab if you understand what i mean. Could this be possible?
